frontend/packages/console-app/src/components/pdb/modals/DeletePDBModal.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { Form } from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { global_warning_color_100 as warningColor } from '@patternfly/react-tokens/dist/js/global_warning_color_100';
+import { useTranslation, Trans } from 'react-i18next';
+import {
+  createModalLauncher,
+  ModalBody,
+  ModalComponentProps,
+  ModalSubmitFooter,
+  ModalTitle,
+} from '@console/internal/components/factory/modal';
+import { LoadingInline } from '@console/internal/components/utils';
+import { PodDisruptionBudgetModel } from '@console/internal/models';
+import { PodDisruptionBudgetKind, k8sKill } from '@console/internal/module/k8s';
+
+const DeletePDBModal: React.FC<DeletePDBModalProps> = ({ close, pdb, workloadName }) => {
+  const [submitError, setSubmitError] = React.useState<string>(null);
+  const [isSubmitting, setIsSubmitting] = React.useState<boolean>(false);
+  const { t } = useTranslation();
+  const pdbName = pdb.metadata.name;
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setIsSubmitting(true);
+    k8sKill(PodDisruptionBudgetModel, pdb)
+      .then(() => {
+        close();
+      })
+      .catch((error) => {
+        setSubmitError(
+          error?.message ||
+            t('console-app~Unknown error removing PodDisruptionBudget {{pdbName}}.', {
+              pdbName,
+            }),
+        );
+      });
+  };
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <div className="modal-content">
```

Comment:
Any reason we need this? I'm not sure it's adding anything to the modal and we should try to move entirely to PF where possible.